daily_milestone.go

```diff
@@ -156,15 +156,33 @@ func getMilestones(baseURL string, token string, projectID string) ([]simpleMile
 }
 
 // CreateMilestoneData creates new milestones with title and due date
-func createMilestoneData(advance int) []simpleMilestone {
+func createMilestoneData(advance int, milestoneTime string) []simpleMilestone {
 	today := time.Now().Local()
 	list := []simpleMilestone{}
 	for i := 0; i < advance; i++ {
-		date := today.AddDate(0, 0, i).Format("2006-01-02")
-		milestone := simpleMilestone{}
-		milestone.Title = date
-		milestone.DueDate = date
-		list = append(list, milestone)
+		if milestoneTime == "daily" {
+			date := today.AddDate(0, 0, i).Format("2006-01-02")
+			milestone := simpleMilestone{}
+			milestone.Title = date
+			milestone.DueDate = date
+			list = append(list, milestone)
+		} else if milestoneTime == "weekly" {
+			date := today.AddDate(0, 0, (i * 7)).Format("2006-01-02")
+			milestone := simpleMilestone{}
+			milestone.Title = date
+			milestone.DueDate = date
+			list = append(list, milestone)
+		} else if milestoneTime == "monthly" {
+			date := today.AddDate(0, i, 0).Format("2006-01-02")
```

Comment:
That would currently lead to 2017-01-01 titles for monthly. In the edge case of it running at the same time as daily this would not create a real monthly milestone.
As above with weekly we most likely need a title of say 2017-01 and a due date of the last day of the month.